real_check/real_check.ps1

```diff
@@ -0,0 +1,57 @@
+#Copyright (c) Microsoft. All rights reserved.
+#Licensed under the MIT license. See LICENSE file in the project root for full license information.
+function Check-Symbols {
+    param(
+        [string] $lib
+    )   
+        try {
+            [string]$output = cmd.exe /c dumpbin /ALL $lib
+        }
+        catch {
+            Write-Error  "Unable to run dumpbin on given lib"
+            return 2
+        }
+        $start_token = "public symbols"
+        $end_token = "Archive member name"
+        try {
+            [int] $start = $output.IndexOf($start_token)
+            [int] $end = $output.IndexOf($end_token, $start)
+            $symbol_text = ($output.Substring($start + $start_token.Length+1, $end-$start-$start_token.Length-1))
+            $split_symbols = $symbol_text.Split('',[StringSplitOptions]::RemoveEmptyEntries)
+        }
+        catch {
+            Write-Error  "Unable to parse dumpbin output"
```

Comment:
>se dumpbin output" [](http://example.com/codeflow?start=39&length=18)

All of these should probably print the lib

Reply:
Done in next iteration.